### ISSUE TITLE
Error: Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by flolege** [inspmoore/rnw_boilerplate#1]
I followed the install instructions and started web with `npm run start-web`. That went well, I can see the Welcome screen in my browser. 

Then I added the following line to the package.json file in the scripts section:
`    "start-android": "react-native run-android",`

and run  `npm run start-android`. The i got the following error in the Metro Bundler window:

Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `C:\Temp\reactnwb\rnw_boilerplate\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:167:1306)
    at ResolutionRequest.resolveDependency (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:80:16)
    at DependencyGraph.resolveDependency (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\node-haste\DependencyGraph.js:237:485)
    at Object.resolve (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\lib\transformHelpers.js:116:25)
    at dependencies.map.result (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\DeltaBundler\traverseDependencies.js:298:29)
    at Array.map (<anonymous>)
    at resolveDependencies (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\DeltaBundler\traverseDependencies.js:294:16)
    at C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\DeltaBundler\traverseDependencies.js:159:33
    at Generator.next (<anonymous>)
    at step (C:\Temp\reactnwb\rnw_boilerplate\node_modules\metro\src\DeltaBundler\traverseDependencies.js:239:307)
 BUNDLE  [android, dev] ..\..\../index.js ░░░░░░░░░░░░░░░░ 0.7% (1/12), failed.

Any hints what could be going on?

**Comment by flolege** [comment]
Maybe I should add that this happens on a Windows 10 environment.
So maybe the solution to this problem is switching to react-native 0.56.1 according to [this issue](https://github.com/facebook/react-native/issues/20331).

I will try it out later. I hope I will not come into dependency hell :)

**Comment by flolege** [comment]
Installing react-native 0.56.1 did not solve the issue, but installing 0.57.4 did!

Just ran `npm install react-native@0.57.4` in root folder.
